db/migrate/20190226151003_create_forms.rb

```diff
@@ -0,0 +1,8 @@
+class CreateForms < ActiveRecord::Migration[5.0]
```

Comment:
What does this do? If it is only adding timestamps, I can't see how it would be useful.